main.cpp

```diff
@@ -49,6 +53,8 @@ int main(int argc, char **argv)
         std::cout << "  -IPATH          Include path." << std::endl;
         std::cout << "  -include=FILE   Include FILE." << std::endl;
         std::cout << "  -UNAME          Undefine NAME." << std::endl;
+        std::cout << "  -std=STD        Specify standard." << std::endl;
+        std::cout << "  -q              Quiet mode (no output)." << std::endl;
```

Comment:
why would you run it in quiet mode if there is no output?

Reply:
Profiling (less code being executed) and crash testing.

Reply:
Also if we would change the exitcode based on if the parsing was successful or not it could use used as a command-line tool and in that case you also might not want *all* the output.